src/main/java/frc/robot/autonomous/Command.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.autonomous;
```

Comment:
The autonomous folder should be on the same directory level as robot and the package declaration should reflect that. For example this line should be change to `package frc.autonomous;`